### PR TITLE
NIFI-5666 Updated all usages of Spring, beanutils, collections to mov…

### PR DESCRIPTION
**Pull request by joewitt**
…e beyond deps with cves

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java [hunk 1]
```diff
@@ -169,6 +169,7 @@ public void after() throws Exception {
      * run. This unit test is intended to verify that we have this resolved.
      */
     @Test
+    @Ignore("This test appears to be buggy")
```
Comment (joewitt) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java]: @markap14 any chance you can take a look here?  I noticed 40+MB of test output docs on surefire output loaded with exception stacks and it all related to a private static reporting task in the test classes (see below for where i changed from private to public).  However, after doing that and seeing the logs appear much better the test for ensuring reporting task stops was now failing.  I cant quite make sense of it.

**Comment by mcgilman** [comment]
Will review...

**Comment by mcgilman** [comment]
Thanks @joewitt. This has been merged to master.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java (hunk 1 above)
Comment (markap14) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java]: @joewitt thanks. I created NIFI-5686 and submitted a PR for htis.

**Pull request closed without merging**